### ISSUE TITLE
get.webgl2.org

### ISSUE THREAD

**Issue by greggman** [KhronosGroup/WebGL#1981]
Should there be a `get.webgl2.org` (or `get.webgl.org/webgl2`) to tell people to download firefox or chrome (or is there any word that Safari and Edge will support WebGL2 anytime soon?)

**Comment by zhenyao** [comment]
From what I heard, khronos is working on it. @kenrussell should know the details.

**Comment by kenrussell** [comment]
Good idea. Actually Khronos isn't working on this at the moment to the best of my knowledge. Any contributions to a get.webgl.org/webgl2/ page would be welcome.

There's no public announcement to make about WebGL 2.0 support in Safari and Edge yet, but all WebGL implementers have been supportive of the direction.
